### PR TITLE
chore: add baoyuan in committer list

### PR DESCRIPTION
**Pull request by Baoyuantop**
Changes:

Add @Baoyuantop on the team page : )
Thanks.

By the way, the `team.js` file is formatted.

### CODE REVIEW

**Review by SkyeYoung — APPROVED** [review]
LGTM 🎉🎉🎉

**Review by yzeng25 — APPROVED** [review]
Congratulations!

**Review by guoqqqi — APPROVED** [review]

**Review by guitu168 — APPROVED** [review]
Congratulations!

**Review by bzp2010 — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Comment by SylviaBABY** [comment]
Congratulation！

File: website/config/team.js [hunk 1]
```diff
@@ -1,276 +1,282 @@
...
+        username: 'bisakh',
+        githubUsername: 'bisakhmondal',
+        avatarUrl: 'https://avatars.githubusercontent.com/u/41498427?v=4',
       },
       {
-        "name": "Bisakh Mondal",
-        "username": "bisakh",
-        "githubUsername": "bisakhmondal",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/41498427?v=4"
+        name: 'Yilin Zeng',
```
Comment (juzhiyuan) [inline website/config/team.js:263]: @Baoyuantop Those formatting changes are made automatically or only your local configuration?

Comment (SkyeYoung) [reply website/config/team.js:263]: Automatically 🤣

Comment (SkyeYoung) [reply website/config/team.js:263]: I haven't had time to push the PR of reformatted code yet.

Comment (juzhiyuan) [reply website/config/team.js:263]: OK

Comment (Baoyuantop) [reply website/config/team.js:263]: We may need a PR to format all the code.

**Pull request merged**